src/sidebar/services/annotations.js

```diff
@@ -51,7 +52,7 @@ export class AnnotationsService {
   /**
    * Extend new annotation objects with defaults and permissions.
    *
-   * @param {AnnotationData} annotationData
+   * @param {Omit<AnnotationData, '$tag'>} annotationData
```

Comment:
I wonder if this deserves an explanatory comment? Or, alternately, some sort of adjustment of the `annotationData` type to explicitly separate out that `$tag` is assigned by the client during initialization? Or...?

Reply:
`AnnotationData` values do have tags in the places where we currently use them. The case where we pass in an `AnnotationData` without a tag is in methods like `createPageNote` and `reply` in this service.